test_unstructured_ingest/expected-structured-output/Sharepoint/SitePages/This-is-a-title.html.json

```diff
@@ -16,7 +16,8 @@
       "filename": "This-is-a-title.html",
       "filetype": "text/html",
       "languages": [
-        "eng"
+        "cat",
+        "fra"
```

Comment:
hmm, so obv not a bug for this PR since ingest is just enabling it, but this one definitely looks like `eng` to me??

`"text": "This is a plain text site page for testing purposes"`

Reply:
@Coniferish might have some better insight?

Reply:
and cc @shreyanid?

Reply:
Yah, `langdetect` doesn't do well with short text, so I added logic in `detect_languages` to default to English, but conservatively set the length to 20 characters. We have a follow-up PR [here](https://github.com/Unstructured-IO/unstructured/issues/1660) and discussed some of that in this Ingest PR: https://github.com/Unstructured-IO/unstructured/pull/1652

Reply:
But this one is actually because language detection happens at the document level by default and the majority of this text is in Latin.